src/test/js/mfa_e2e_test.js

```diff
@@ -653,3 +653,111 @@ Scenario('@functional @mfaLogin As a user, I can login to the mfa turned off ser
 
     I.resetRequestInterception();
 });
+
+Scenario('@functional @mfaLogin @mfaSkipStepUpLogin As a user, I can login to the MFA turned on service and then authorize to the same service with prompt=login and login again', async ({ I }) => {
+    const nonce = "0km9sBgZfnXv8e_O7U-XmSR6vtIgsUVTXybVUdoLV7g";
+    const loginUrl = `${TestData.WEB_PUBLIC_URL}/login?redirect_uri=${mfaTurnedOnService1.activationRedirectUrl}&client_id=${mfaTurnedOnService1.oauth2ClientId}&state=44p4OfI5CXbdvMTpRYWfleNWIYm6qz0qNDgMOm2qgpU&nonce=${nonce}&response_type=code&scope=${scope}&prompt=`;
+
+    I.amOnPage(loginUrl);
+    I.waitForText('Sign in');
+    I.fillField('#username', mfaUserEmail);
+    I.fillField('#password', userPassword);
+    I.interceptRequestsAfterSignin();
+    I.click('Sign in');
+    I.seeInCurrentUrl("/verification");
+    I.waitForText('Verification required');
+    const otpCode = await I.extractOtpFromNotifyEmail(mfaUserEmail);
+
+    I.fillField('code', otpCode);
+    I.click('Continue');
+    I.waitForText(mfaTurnedOnService1.activationRedirectUrl.toLowerCase());
+    I.see('code=');
+    I.dontSee('error=');
+
+    const mfaturnedOnService1PageSource = await I.grabSource();
+    const mfaturnedOnService1Code = mfaturnedOnService1PageSource.match(/\?code=([^&]*)(.*)/)[1];
+    const mfaturnedOnService1AccessToken = await I.getAccessToken(mfaturnedOnService1Code, mfaTurnedOnService1.oauth2ClientId, mfaTurnedOnService1.activationRedirectUrl, serviceClientSecret);
+
+    let mfaturnedOnService1JwtDecode = await jwt_decode(mfaturnedOnService1AccessToken);
+
+    assert.equal("access_token", mfaturnedOnService1JwtDecode.tokenName);
+    assert.equal(nonce, mfaturnedOnService1JwtDecode.nonce);
+    assert.equal(1, mfaturnedOnService1JwtDecode.auth_level);
+
+    I.amOnPage(loginUrl);
+    const idamSessionCookie = await I.grabCookie('Idam.Session');
+    const cookie = idamSessionCookie.value;
+
+    const authLocation = await I.getWebPublicOidcAuthorizeWithLogin(mfaTurnedOnService1.oauth2ClientId, mfaTurnedOnService1.activationRedirectUrl, scope, nonce, cookie);
+    expect(authLocation).to.includes(`/login?client_id=${mfaTurnedOnService1.oauth2ClientId}&redirect_uri=${mfaTurnedOnService1.activationRedirectUrl}`);
```

Comment:
Should we add prompt=login to the expected URL?

Reply:
That actually assumes we add it to the login page URL?

Reply:
No - it doesn't look like it does get added to the login url that the api sends back as the redirect. I guess it's redundant at that point anyway. I have had to fix the test to make sure it uses the ${TestData.WEB_PUBLIC_URL} value instead of the one that the api sends back (so that we keep using the PR version of web public)